### ISSUE TITLE
Large Aerospace unit construction

### ISSUE THREAD

**Issue by monbvol** [MegaMek/megameklab#192]
The current drag/drop and right click context menu options are rather cumbersome, especially for allocating truly bulk assignments.

As such I would recommend when allocating weapons add a check box with the firing arcs and a dialogue to adjust ammunition for weapons that need it.  Ideally automatically duplicating the setup to each arc/bay selected.

**Comment by BLR-IIC** [comment]
One of the things I noticed when allocating weapons is that you can select multiple weapons, then drag and drop them into the desired firing arc.  It's not obvious unless you misclick a few times.

**Comment by monbvol** [comment]
I know you can do that but it still gets rather tedious with really bulk
allocation, especially if they require ammo.

**Comment by BLR-IIC** [comment]
I agree it can use improvement.  It would be nice for the firing arcs that require symmetry to automate placement of weapons.  For an example, if I add two of five NL55 in my weapons list to the left broadside arc, MML would add two of the remaining NL55 in the weapons list to the right broadside arc.

**Comment by lenny53207** [comment]
It's really terrible for ammo, it would be nice to click a bay and enter the number of shots it has, then duplicate it.  I just revamped a Warship with 6600 rounds of ammo for it's 55 NAC bays and allocation those shots with the click on ammo, click arc, click bay 165 times was very tedious
